src/rendering/renderable.cpp

```diff
@@ -208,7 +208,7 @@ bool Renderable::isEnabled() const {
 }
 
 void Renderable::onEnabledChange(std::function<void(bool)> callback) {
-    _enabled.onChange([&] () {
+    _enabled.onChange([this, callback] () {
```

Comment:
Not sure if it is possible:

`_enabled.onChange([this, c = std::move(callback)] () { c(isEnabled()); });`